Bejebeje.Domain/ArtistImage.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Bejebeje.Domain
+{
+  using System;
+
+  public class ArtistImage
+  {
+    public int Id { get; set; }
+
+    public byte[] Data { get; set; }
+
+    public DateTime CreatedAt { get; set; }
+
+    public DateTime? ModifiedAt { get; set; }
+
+    public int ArtistId { get; set; }
```

Comment:
Shouldn't this be a foreign key? (or did i miss it and you defined that in code?) (this is why i prefer using the Attributes, but personal choice ofc)